.github/workflows/assigner.yml

```diff
@@ -51,14 +60,15 @@ jobs:
         FLAGS+=" -r ${{ github.event.repository.name }}"
         FLAGS+=" -M MAINTAINERS.yml"
         if [ "${{ github.event_name }}" = "pull_request_target" ]; then
-            FLAGS+=" -P ${{ github.event.pull_request.number }}"
+          FLAGS+=" -P ${{ github.event.pull_request.number }} --updated-manifest pr_west.yml"
+          python3 scripts/set_assignees.py $FLAGS
         elif [ "${{ github.event_name }}" = "issues" ]; then
-            FLAGS+=" -I ${{ github.event.issue.number }}"
+          FLAGS+=" -I ${{ github.event.issue.number }}"
+          python3 scripts/set_assignees.py $FLAGS
         elif [ "${{ github.event_name }}" = "schedule" ]; then
-            FLAGS+=" --modules"
+          FLAGS+=" --modules"
+          python3 scripts/set_assignees.py $FLAGS
         else
           echo "Unknown event: ${{ github.event_name }}"
           exit 1
         fi
-
-        python3 scripts/set_assignees.py $FLAGS
```

Comment:
why is this moved? btw would be cool to add a a `-u`  so you get unbuffered output on the action, not that it matters much here since this runs fast but why not...